bioblend/_tests/TestGalaxyJobs.py

```diff
@@ -1,10 +1,77 @@
+import os
+from datetime import datetime, timedelta
+
+from bioblend import galaxy
 from . import (
     GalaxyTestBase,
     test_util
 )
 
 
 class TestGalaxyJobs(GalaxyTestBase.GalaxyTestBase):
+    @test_util.skip_unless_tool("random_lines1")
+    def test_get_jobs(self):
+        user1 = self.gi.users.create_local_user('user1', 'email1@email.test', 'password1')
+        user2 = self.gi.users.create_local_user('user2', 'email2@email.test', 'password2')
+        key1 = self.gi.users.create_user_apikey(user1['id'])
+        key2 = self.gi.users.create_user_apikey(user2['id'])
+        user1_gi = galaxy.GalaxyInstance(url=self.gi.base_url, key=key1)
+        user2_gi = galaxy.GalaxyInstance(url=self.gi.base_url, key=key2)
```

Comment:
Can you just test using the default user instead of adding others? I think you can still filter by tool if you also filter by history.

Reply:
Okay, got it. They have been removed.